packages/api/src/module/write/email-confirmation/domain/events.ts

```diff
@@ -0,0 +1,3 @@
+import { EmailConfirmationWasRequested } from '@/module/events/email-confirmation-was-requested.domain-event';
+
+export type RequestEmailConfirmationDomainEvent = EmailConfirmationWasRequested;
```

Comment:
The purpose of this type union is to group together all events from one stream (related by some business rule - like `you cannot approve not requested confirmation` etc.), so better name would be `EmailConfirmationDomainEvent`. Currently there is only one `EmailConfirmationWasRequested`, but next `EmailConfirmationWasApproved` will be part of this union.